Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+                  <ul>
+                    <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                    <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                    <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                  </ul>
+                </div>
+                  <button><a class="boton2">See Project</a></button> 
+                </div>
+        
+                <div class="projects">
+                  <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+                  <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="list_programs2">
+                    <ul>
+                      <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                      <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                      <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                    </ul>
+                  </div>
+                    <button><a class="boton2">See Project</a></button> 
+                  </div>
+
+    </section>
+
+  <section class="page3">
+    <h1 class="about">About me</h1>
+    <p class="parrafoAM">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <button><a class="boton3">Get My Resume</a></button> 
+    <img class="vector2"src="./images/Vector.svg">
+    <div class="skills">
+      <ul>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-diamond"></i>     Languaje</li>
+          <li class="smallskill">JavaScript</li>
+          <li class="smallskill">Ruby</li> 
+          <li class="smallskill">Html</li>
+          <li class="smallskill">Css</h4></li> 
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-square"></i>     Frameworks</li>
+           <li class="smallskill">Bootstrap</li>
+           <li class="smallskill">Ruby_on_Rails</li> 
+           <li class="smallskill">RSPec</li>  
+           <li class="smallskill">CapyBara</li>
+           <li class="smallskill">Selenium</li>
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-circle"></i>     Skills</li>
+           <li class="smallskill">Codekit</li>
+           <li class="smallskill">Github</li> 
+           <li class="smallskill">Codepen</li>
+           <li class="smallskill">Gitlab</li>
+           <li class="smallskill">Terminal</li>  
+      </ul>
+    </div>
```

Comment:
- [ ] Consider using a nested list for this section as demanded by the project requirements. A nested list will look like this;
```html
<ul>
  <li>
    Languages
    <ul>
      <li>CSS</li>
      ...
    </ul>
  </li>
  <li>
    Frameworks
    <ul>
      <li>Bootstrap</li>
      ...
    </ul>
  </li>
  <li>
    Skills
    <ul>
      <li>Codekit</li>
      ...
    </ul>
  </li>
</ul>
```

- [ ] I noticed that your list of technologies is shifted to the right, Kindly align them to the center to match the design template. Try using `padding-inline-start: 0;` on the `ul` element wrapping it.
### Your design
![3](https://user-images.githubusercontent.com/94551484/198818178-ab4c216e-4def-4241-86fe-81afb005a23d.png)

### Figma template
![f3](https://user-images.githubusercontent.com/94551484/198818180-85b1715c-f310-4c85-89b4-c2fbbd42c317.png)